developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
+|                               |                                                  | | ``3`` = Public link                                                    |
+|                               |                                                  | | ``4`` = Email                                                          |
+|                               |                                                  | | ``6`` = Federated cloud share                                          |
+|                               |                                                  | | ``7`` = Circle                                                         |
+|                               |                                                  | | ``8`` = Guest                                                          |
+|                               |                                                  | | ``9`` = Remote group                                                   |
+|                               |                                                  | | ``10`` = Talk conversation                                             |
+|                               |                                                  | | ``12`` = Deck                                                          |
+|                               |                                                  | | ``15`` = Science mesh                                                  |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <ocs:share-permissions />     | | The permissions that the                       | | ``1`` = Read                                                           |
+|                               | | user has over the share.                       | | ``2`` = Update                                                         |
+|                               |                                                  | | ``4`` = Create                                                         |
+|                               |                                                  | | ``8`` = Delete                                                         |
+|                               |                                                  | | ``16`` = Share                                                         |
+|                               |                                                  | | ``31`` = All                                                           |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <ocm:share-permissions />     | | The permissions that the                       | ``share``, ``read``, ``write``                                           |
```

Comment:
The output is a json array:
https://github.com/nextcloud/server/blob/278f6413c9ee6d5c011c59a81d2f8e24a86dfbac/apps/dav/lib/Connector/Sabre/FilesPlugin.php#L322